dts/bindings/net/wireless/nordic,nrf-radio.yaml

```
@@ -0,0 +1,103 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Nordic DFE antennas matrix description
+
+    dfegpio[n]-gpios properties provide GPIO pins to Radio peripheral. Provided
+    GPIO pins are used to drive antennas switching by Radio Direction Finding
+    Extension. E.g. to use 12 antennas 4 GPIO pins must be provided,
+    4 GPIO pins support switching up to 16 antennas.
+
+    All dfegpio[n]-gpios properties have required property set to false.
+    It depends on end user which Radio PSEL_GPIO[n] registers will be used
+    to drive antenna switches.
+    Nevertheless there is a requirement that at least two antennas are
+    available. That means at least one dfegpio[n]-gpios property must be
+    provided, to give Radio possibility to switch between these two antennas.
```

```suggestion
    The dfegpio[n] properties are used to initialize the PSEL.DFEGPIO[n] registers.
    None of the dfegpio[n]-gpios properties are marked required, since
    which Radio PSEL.DFEGPIO[n] registers will be used to drive antenna switches
    depends on the use case.
    
    Nevertheless, at least two antennas must be available. That means at
    least one dfegpio[n]-gpios property must be provided, to give the radio
    the possibility to switch between two antennas.
```

This is just some suggested rewording. I'm also suggesting using the register names in the product specification. I am not trying to change the meaning of your description.

---

Changed according to suggestion